src/app/shared/filesystem/file-list-view/components/file-list-item/file-list-item.component.html

```diff
@@ -1,31 +1,34 @@
-<div class="grid-row wrapper">
+<div class="grid-row wrapper" role="link" tabindex="0" attr.aria-labelledby="name description updated size">
   <div
     (click)="handleClick($event)"
     class="text-contained inline-flex"
     [ngClass]="{ 'hover-react': previewUrl }"
     tip="{{ file?.name }}"
     tipLocation="above"
+    id="name"
   >
     <i class="fas {{ icon }} icon"></i>
     {{ file?.name }}
     <span class="external-link-icon" *ngIf="previewUrl"
       ><i class="far fa-external-link"></i
     ></span>
   </div>
-  <div class="text-contained description">
+  <div class="text-contained description" id="description">
```

Comment:
Since this is a component, the id property here should be more specific since we can't guarantee that there won't be any other elements rendered to the page with the same id.

Reply:
Resolved with 0f43c62